README.md

```diff
@@ -1,2 +1,155 @@
-# ToDolistsm
-In this project, you will build a simple HTML list of To Do tasks. The list will be styled according to the specifications listed later in this lesson. This simple web page will be built using webpack and served by a webpack dev server.
+# To do List
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 TO DO list<a name="about-project"></a>
+
+**To do list ** is developed in HTML, CSS, JavaScript, and Es 6 modules,webback
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>
+      HTML <br>
+      CSS
+      </a>
+    </li>
+  </ul>
+</details>
+
+
+##
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Brake down the script into four modules**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project, you need:
+
+### Setup
+
+Clone this repository to your desired folder
+https://github.com/samuelkinyanjui/todolistsm)
+
+
+### Install
+
+Install this project with: 
+
+```sh
+  cd Capstone-Project_sam
+ npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+> npm install
+
```

Comment:
**Incorrect command:** In the Usage section, there is an inconsistency between the provided command and the description. The description mentions running the project with the command `npm install`, which is incorrect. It should be something like `npm run build` and `npm start` instead.